docs/vs-tools-tizen/how-to-install.md

```diff
@@ -1,129 +1,138 @@
+     1. Launch a Tizen tool that is not yet installed (such as Emulator Manager or Package Manager). In the window that pops up, select **Use installed Tizen SDK**.
 
-### Check Nuget server for build task
-   * Check the package server in Tools > Option > NuGet Package Manager > Package Sources
-     > https://tizen.myget.org/F/dotnet/api/v3/index.json
+        ![Baseline SDK Install](media/howtoinstall-baselineinstall5.png)
+     2. Enter the root directory of your existing Tizen Studio installation.
 
-### Cannot remove Extension
-   * Shutdown all instances of Visual Studio
-   * Go to VS 2017 directory and order by time in explorer
-   * After checking the folder, Remove Tizen Extension
-   * Open a developer command prompt as administrator
-   * Execute : "devenv /setup"
+        ![Baseline SDK Install](media/howtoinstall-baselineinstall6.png)
+     3. If the installer gives a warning about your Tizen Studio version being too low, update the Tizen Studio by using the Package Manager after setting the tool path.
+
+   - If Tizen Studio is not installed:
+     1. Launch a Tizen tool that is not yet installed (such as Emulator Manager or Package Manager). In the window that pops up, select **Install new Tizen SDK**.
+
+        ![Baseline SDK Install](media/howtoinstall-baselineinstall1.png)
+     2. Follow the instructions given by the Install Wizard (Review the EULA and set the installation path).
+     3. In the last step of the Install Wizard, click **Install**.
+
+        The installer is downloaded and the baseline SDK is installed automatically.
+
+        ![Baseline SDK Install](media/howtoinstall-baselineinstall2.png)
+     4. You can install additional packages (such as the TV Emulator image and SDK tools) through the Package Manager.
+
+        ![Baseline SDK Install](media/howtoinstall-baselineinstall4.png)
+
+
+## Troubleshooting
+
+If you encounter problems with the installation:
+
+- To check whether VSIX is installed correctly:
+  - Check the Visual Studio Tools for Tizen properties by selecting **Tools &gt; Extensions and Updates** and searching for "Tizen".
+
+    ![Check VSIX](media/cps-extension-and-updates.png)
+
+- To make sure the baseline Tizen SDK is installed correctly:
+  - Check the tool path by selecting **Tools &gt; Options &gt; Tizen &gt; Tools**.
+
+    ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
+
+- To check whether Tizen Project properties have been copied correctly:
+  - Make sure that the `Tizen.NET.ProjectType.props` and `Tizen.NET.ProjectType.targets` files are located in the `C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Tizen\VisualStudio` folder.
+
+     ![Project Property](media/cps-project-property.png)
+
+- To check the Nuget server for build tasks:
+  - Check the package server in **Tools &gt; Option &gt; NuGet Package Manager &gt; Package Sources &gt; https://tizen.myget.org/F/dotnet/api/v3/index.json**.
+
+- If you cannot remove the Tizen extension:
```

Comment:
The step list makes no sense. What exactly needs to be done in the Visual Studio directory after organizing the files based on the timestamp? What needs to be checked in the folder?

Reply:
Removed